client/packages/system/src/ContactTrace/DetailView/DetailView.tsx

```diff
@@ -0,0 +1,270 @@
+import React, { FC, useEffect, useState } from 'react';
+import {
+  useTranslation,
+  DetailViewSkeleton,
+  AlertModal,
+  useNavigate,
+  RouteBuilder,
+  useDebounceCallback,
+  useBreadcrumbs,
+  useFormatDateTime,
+  Breadcrumb,
+  useIntlUtils,
+  ContactTraceNodeStatus,
+} from '@openmsupply-client/common';
+import {
+  useJsonForms,
+  useContactTraces,
+  ContactTraceRowFragment,
+  useDocumentRegistry,
+  SchemaData,
+} from '@openmsupply-client/programs';
+import { AppRoute } from '@openmsupply-client/config';
+import { Toolbar } from './Toolbar';
+import { Footer } from './Footer';
+import { PatientTabValue } from '../../Patient/PatientView/PatientView';
+import { usePatient } from '../../Patient';
+
+export type ContactTrace = {
```

Comment:
I moved the useContactTraceData hook into its own file since the data types belongs to it and it would be hard to understand what the types are used for otherwise...